src/index.js

```diff
@@ -35,6 +34,7 @@ export {default as NavigationControl} from './components/navigation-control';
 
 // Utilities
 // export {default as fitBounds} from './utils/fit-bounds';
+export {default as MapControls} from './utils/map-controls';
```

Comment:
Thanks @ibgreen for catching this. Yes, if we'd like to export MapControls, we should at lease implement it as a subclass of Controls so that user can extend the Controls class to create OrbitControls, FreeCameraControls, FPSControls, etc....

And I also think MapControls should really be MapViewControl or MapViewportControl